src/routes/docs/tooling/command-line/non-interactive/+page.markdoc

```diff
@@ -8,16 +8,31 @@ The Appwrite CLI can be used in a non-interactive and headless manner, without s
 ```sh
 appwrite client \
     --endpoint https://cloud.appwrite.io/v1 \
-    --project-id [YOUR_PROJECT_ID] \
+    --project-id [PROJECT_ID] \
     --key YOUR_API_KEY
 ```
 
 When you set the global configuration parameters using the `appwrite client` command, they take precedence over the local configuration parameters in your `appwrite.json` thereby switching the CLI to non-interactive mode.
 
 In this mode, the CLI can only interact with one project at a time.
 
+Use `appwrite login` with your email, password, and OTP to log in so you can run tests or set up project-per-branch previews, or you log in using an API key.
+
+```sh
+appwrite login --email "<EMAIL>" --password "<PASSWORD>" --mfa totp  
```

Comment:
I would not mention MFA in this example. If we want to, we can, but I would do it in separate snippet. And this is the correct syntax:

```
appwrite login --email "<EMAIL>" --password "<PASSWORD>" --mfa totp  --code "<CODE>"
```

--mfa could also be email, recovery code, ...

Reply:
Let's not mention MFA because it's really impractical to use MFA non-interactively. You can't use email or phone because the create MFA challenge call must happen first to send out the code. TOTP would require you to have the private key in CI and somehow generate the TOTP, which isn't typically how it's done.